app/javascript/stores/components/easy-product-form.vue

```diff
@@ -0,0 +1,226 @@
+<template>
```

Comment:
Definition for rule 'vue/max-len' was not found  vue/max-len

Reply:
Lo definimos por todas partes y no hubo caso, así que lo dejamos pasar.